server/testdata/server.crt

```diff
@@ -0,0 +1,25 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
BTW @timvaillancourt 
You can generate the test file when running the unit test.

```
fun genCRTTest(t *testing.T) {
    // do something just like echo "data" >> server.crt
}

func cleanCRTtest(t *testing.T) {
    // do something like rm
}
```